kippenhahn/kippenhahn.py

```diff
@@ -991,7 +1014,10 @@ def plot(self, ax=None):
         if self._param['signed_log_cmap']:
             data_to_plot = np.sign(np.transpose(self._data)) * np.log10(np.maximum(1.,np.abs(np.transpose(self._data))))
         else:
-            data_to_plot = np.log10(np.transpose(self._data))
+            if self._param['log_cmap']:
+                data_to_plot = np.log10(np.abs(np.transpose(self._data)))
+            else:
+                data_to_plot = np.transpose(self._data)
```

Comment:
Just a matter of style, should you better use the structure:
```
if self._param['signed_log_cmap']:
    ...
elif self._param['log_cmap']:
    ...
else:
    ...
```
instead of
```
if self._param['signed_log_cmap']:
    ...
else:
    if self._param['log_cmap']:
        ...
    else:
        ...
```